### DISCUSSION TITLE
Do you have any plan to support github discussion APIs in the future release?

### DISCUSSION THREAD

**Discussion by donggyu04** [hub4j/github-api#1056] [Q&A] [answered]
Hi, thanks for support the nice library.
My simple question is, Is there any plan to support [discussion API](https://github.com/github/feedback/discussions/43#discussioncomment-399047)(not team's discussion)?
> The latest version(currently 1.123) seems not to support it.

**Comment by bitwiseman** [comment, marked as answer]
It looks like there is only a GraphQL endpoint for this, which will make implementation slower because we don't have any other uses of GraphQL in this library yet.  
https://docs.github.com/en/graphql/guides/using-the-graphql-api-for-discussions

Feel free to file an issue.  If you're interested in implementing it, I'm happy to make suggestions.

**Reply by sundergopalsingh** [reply]
So is there a way we can utilize graphql apis provided by github as per your comment it does not seem like we support it in this library. 
If not any plans to support in the future as I would say it provides flexibility to decrease the number of calls to github
@bitwiseman

**Reply by bitwiseman** [reply]
GraphQL support is tracked as #521 , however it is a very large undertaking. In particular, designing to take advantage of that flexibility.  Also, the paginated results in graphQL are handled in a different (and better) way than REST API, so we need that implemented before we can do even basic querying. 

So, it is on the list but no one has stepped up to do the work.  Would you like to take that on?
